docs/client.md

```diff
@@ -148,7 +148,7 @@ Given the following line now added to `eik.json`
 
 When we run `eik publish` any "bare imports" refering to either `lit-html` or `lodash` will be mapped to the URLs in our map.
 
-In this way, you can control which version of `react` or `lit-html` or `lodash` all your apps are using. In combination with package `alias` URLs, you have a powerful way to manage key shared dependencies for your apps in production without the need to redeploy or rebundle when a new version of a dependency is released.
+In this way, you can control which version of `react` or `lit-html` or `lodash` all your apps are using. In combination with package `alias` URLs, you have a powerful way to manage key shared dependencies for your apps in production without the need to redeploy or rebundle when a new version of a dependency are released.
```

Comment:
A new version of a dependency are released... *are* doesn't sound right to me, are you sure?

Reply:
You're right, this is a bad change, will fix.